addons/mail/static/src/core/common/thread.xml

```diff
@@ -57,7 +57,7 @@
             </div>
         </t>
     </div>
-    <t t-if="!env.inChatter" t-call="mail.Thread.jumpPresent"/>
+    <t t-if="!env.inChatter and !props.thread.isEmpty" t-call="mail.Thread.jumpPresent"/>
```

Comment:
It should be handled in `JumpToPresent` template with other conditions which are responsible for the display of this banner.